app/assets/stylesheets/all/elmo_forms.css.scss

```diff
@@ -233,20 +233,10 @@ div.cascading-selects {
       border: none;
     }
 
-    div.label_and_control {
```

Comment:
I did check the modals when i implemented, but just to make sure i decided to check them again. I searched for modal classes on views of the project and checked them on the app and they are all being displayed fine. Will merge. Thanks!